workspace/workspace.go

```diff
@@ -216,11 +217,15 @@ func loadWorkspaceFromConfigYaml(configFile string, path string) (Workspace, err
 }
 
 func LoadWorkspace() (Workspace, error) {
+	return loadWorkspace()
+}
```

Comment:
What's the purpose of having both an exported and a non-exported version of this function?

Reply:
There's still other places where the exported version is used. I'd like to change this, for the time being, I'll keep it like that.